src/main/scala/cromwell/logging/TerminalLayout.scala

```diff
@@ -7,37 +7,38 @@ import ch.qos.logback.classic.Level
 import ch.qos.logback.classic.pattern.ThrowableProxyConverter
 import ch.qos.logback.classic.spi.ILoggingEvent
 import ch.qos.logback.core.LayoutBase
-import cromwell.logging.TerminalLayout.EnhancedILoggingEvent
 import cromwell.util.TerminalUtil
 
+object TerminalLayout {
+  val Converter = new ThrowableProxyConverter
+  Converter.start()
+
+  implicit class EnhancedILoggingEvent(val event: ILoggingEvent) extends AnyVal {
+    def toStackTrace: String = Converter.convert(event)
+  }
+
+  implicit class ColorString(msg: String) {
+    def colorizeUuids: String = {
+      "UUID\\((.*?)\\)".r.findAllMatchIn(msg).foldLeft(msg) {
+        case (l, r) =>
+          val color = (Math.abs(17 * r.group(1).substring(0,8).map(_.toInt).product) % 209) + 22
+          l.replace(r.group(0), TerminalUtil.highlight(color, r.group(1)))
+      }
+    }
+    def colorizeCommand: String = msg.replaceAll("`([^`]*?)`", TerminalUtil.highlight(5, "$1"))
+  }
+}
+
```

Comment:
As an aside, would it be possible to make the colors less IN YOUR FACE, bright & bold? that's 90% of why I hate them so much